modules/admin_manual/pages/troubleshooting/path_filename_length.adoc

```diff
@@ -0,0 +1,102 @@
+= Path and Filename Length Limitations
+:toc: right
+:fs-limits-url: https://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits
+:enable-long-paths-url: https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd#enable-long-paths-in-windows-10-version-1607-and-later
+:getconf-url: http://manpages.ubuntu.com/manpages/focal/man1/getconf.1.html
+
+== Introduction
+
+Depending on the underlaying filesystem of a mount point, the maximum length of a path component and the filename can differ. This is important if you start copying or moving single files or even complete paths from one mount to another where the target mount has a more restrictive lengt rule than the source. This can also be an issue when using a synchronisation client when using an Operating System (OS) having a different filesystem than the source mount filesystem has. The following table should give you a brief overview as guideline.
```

Comment:
```suggestion
Depending on the underlying filesystem of a mount point, the maximum length of a path component and the file name can differ. This is important if you start copying or moving single files or even complete paths from one mount to another where the target mount has a more restrictive length rule than the source. This can also be an issue when using a synchronization client running on an Operating System (OS) with a different filesystem than the source mount filesystem. The following table gives you a brief overview as a guideline.
```